block-storage/src/main/scala/coop/rchain/blockstorage/dag/BlockDagKeyValueStorage.scala

```diff
@@ -322,4 +339,47 @@ object BlockDagKeyValueStorage {
       stores.invalidBlocks,
       stores.equivocations
     )
+
+  def migrateLfb[F[_]: Sync: Log](
```

Comment:
This function should be defined on `LastFinalizedKeyValueStorage` instance which is used in Setup and will be deleted with it.

Reply:
Makes sense, thanks

Reply:
I found it more simple to have it in BlockDagStorage, as I'm not constrained with API. But ok, I will move it to finalized storage.

Reply:
Actually its hard to do that in LastFinalizedKeyValueStorage, because API that BlockDagStorage offers is to write the whole finalization at once https://github.com/rchain/rchain/pull/3459#discussion_r662431601, so it requires calling representation. And representation cannot be created because there is no lastFinalizedBlock (as it has not been migrated)